### DISCUSSION TITLE
[Feature] - Hide posts from list/search/previous/next using front matter

### DISCUSSION THREAD

**Discussion by kamasylvia** [CaiJimmy/hugo-theme-stack#1073] [Ideas]
I'd like to request a feature that allows authors to hide specific posts from the main blog list while still enabling access through direct links. 

Desired Functionality:

- Hidden posts should not be included in blog listings, search results, or sitemaps.
- Users who directly access a hidden post's URL should still be able to view the content.

Similar request I found:

- [Howto hide from menu using front matter](https://discourse.gohugo.io/t/howto-hide-from-menu-using-front-matter/18960)

- [Howto: Hidden pages ( no links, no search)](https://discourse.gohugo.io/t/howto-hidden-pages-no-links-no-search/16194)

Similar Existing Solutions:

- Hexo: The hexo-hide-posts plugin (https://github.com/prinsss/hexo-hide-posts) offers a similar solution for Hexo blogs.

Inspiration from Existing Solutions:

- [Creating Unlisted Content in Hugo](https://bphogan.com/2020/08/11/2020-08-11-creating-unlisted-content-in-hugo/) provides valuable insights on implementing unlisted content in Hugo.
- [hugo-theme-learn](https://github.com/matcornic/hugo-theme-learn/blob/master/exampleSite/content/cont/pages/_index.en.md#front-matter-configuration) demonstrates potential approaches for managing hidden content.

**Comment by CaiJimmy** [comment]
This is already supported by Hugo itself: https://gohugo.io/content-management/build-options/
